app/views/providers/confirm_client_declarations/show.html.erb

```diff
@@ -8,7 +8,7 @@
   <%= page_template page_title: t(".h1-heading"), form: do %>
 
     <% provider_firm_name = @legal_aid_application.provider.firm.name %>
-    <% if @legal_aid_application.non_means_tested? %>
+    <% if @legal_aid_application.applicant&.no_means_test_required? %>
```

Comment:
I'm happy with it regardless but would it make more sense to use `under_18?` rather than `no_means_test_required?` because there's no means test if they're over 18 too with SCA, right?